docs/installing/adding_devices.md

```diff
@@ -30,13 +30,13 @@ The following instructions guide you through the process of installing the requi
 
 - x86_64
   - {{site.data.keyword.linux_bit_notm}} devices or virtual machines that run Ubuntu 24.x (noble), Ubuntu 22.x (jammy), Ubuntu 20.x (focal), Ubuntu 18.x (bionic), Debian 12 (bookworm), Debian 11 (bullseye), Debian 10 (buster), Debian 9 (stretch)
-  - {{site.data.keyword.rhel}} 8.1 - 8.5 (via Docker), 8.6 - 8.8 and 9.0 - 9.4 (via Podman 4.x).  See Notes.
-  - {{site.data.keyword.fedora}} Workstation 36 - 40
+  - {{site.data.keyword.rhel}} 8.1 - 8.5 (via Docker), 8.6 - 8.8 and 9.0 - 9.5 (via Podman 4.x).  See Notes.
+  - {{site.data.keyword.fedora}} Workstation 36 - 43
```

Comment:
Fedora 41 is the current release. Fedora 42 is planned for April 2025.

Reply:
Thanks... Fixed